### DISCUSSION TITLE
Welcome to Harbor Discussions!

### DISCUSSION THREAD

**Discussion by nkomarn** [nkomarn/Harbor#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MasterSquinch** [comment]
Do you have any plans to port to fabric?

**Reply by nkomarn** [reply]
Not currently, but I potentially can if enough people would like to use Harbor on Fabric.

**Reply by MasterSquinch** [reply]
Oh god I'd love to, and I wouldn't mind helping I'm just not sure where to begin. 

My favorite sleep overhaul by a mile.
